app/serializers/post_revision_serializer.rb

```diff
@@ -25,11 +25,9 @@ class PostRevisionSerializer < ApplicationSerializer
              :title_changes,
              :user_changes,
              :tags_changes,
-             :wiki,
+             :category_id_changes,
```

Comment:
We don't need to know whether a post is currently a `wiki`. We're only interested in the changes. Which is already handled by the `add_compared_field :wiki` below.

But we do need to filter any category changes the current user can't see 🙈